easybuild/easyconfigs/m/magma/make.inc

```diff
@@ -0,0 +1,49 @@
+#//////////////////////////////////////////////////////////////////////////////
+#   -- MAGMA (version 1.6.1) --
+#      Univ. of Tennessee, Knoxville
+#      Univ. of California, Berkeley
+#      Univ. of Colorado, Denver
+#      @date January 2015
+#//////////////////////////////////////////////////////////////////////////////
+
+# GPU_TARGET contains one or more of Tesla, Fermi, or Kepler,
+# to specify for which GPUs you want to compile MAGMA:
+#     Tesla  - NVIDIA compute capability 1.x cards (no longer supported in CUDA 6.5)
+#     Fermi  - NVIDIA compute capability 2.x cards
+#     Kepler - NVIDIA compute capability 3.x cards
+# The default is "Fermi Kepler".
+# See http://developer.nvidia.com/cuda-gpus
+#
+#GPU_TARGET ?= Fermi Kepler
+GPU_TARGET = Kepler
+
+CXX       = CC
+CC        = cc
+NVCC      = nvcc
+FORT      = ftn         
+
+ARCH      = ar
+ARCHFLAGS = cr
+RANLIB    = ranlib
+
+# Use -fPIC to make shared (.so) and static (.a) library;
+# can be commented out if making only static library.
+FPIC      = -fPIC
```

Comment:
note that EB has its own switch for PIC builds; AFAIK, this L31 would prevail over L10 on the easyconfig above;
probably no-op for you, just take a note